docs/eventing/channels/subscriptions.md

```diff
@@ -37,16 +61,11 @@ After you have created a channel and a sink, you can create a subscription to en
     &nbsp;
     - `--sink-reply` and `--sink-dead-letter` are optional arguments. They can be used to specify where the sink reply will be sent to, and where to send the cloud event in case of a failure, respectively. Both use the same naming conventions for specifying the sink as the `--sink` flag.
 
-    Example command:
-    ```
-    kn subscription create mysubscription --channel mychannel --sink ksvc:myservice
-    ```
-
-    This example command creates a channel named `mysubscription`, that routes events from a channel named `mychannel` to a Knative service named `myservice`.
-
-    **NOTE:** The sink prefix is optional. It is also possible to specify the service for `--sink` as just `--sink <service_name>` and omit the `ksvc` prefix.
-
+This example command creates a channel named `mysubscription` that routes events from a channel
+named `mychannel` to a Knative service named `myservice`.
 
+**NOTE:** The sink prefix is optional. You can also specify the service for `--sink` as just
+`--sink <service_name>` and omit the `ksvc` prefix.
```

Comment:
formatting for mkdocs?

Reply:
also `<service-name>` - watch formatting for the variables and make sure they're consistent throughout